src/pymovements/gaze/_utils/parsing.py

```diff
@@ -416,10 +426,9 @@ def _pre_process_metadata(metadata: defaultdict[str, Any]) -> dict[str, Any]:
         metadata['version_1'], metadata['version_2'],
     )
 
-    if 'resolution' in metadata:
-        coordinates = [int(coord) for coord in metadata['resolution'].split()]
-        resolution = (coordinates[2] - coordinates[0] + 1, coordinates[3] - coordinates[1] + 1)
-        metadata['resolution'] = resolution
+    if 'DISPLAY_COORDS' in metadata:
+        display_coords = tuple(float(coord) for coord in metadata['DISPLAY_COORDS'].split())
```

Comment:
the display coords are always integers, aren't they? if so, then cast to `int` instead.

You can resolve this comment by yourself to auto-merge.